### PR TITLE
Migrate to Python3

### PR DESCRIPTION
**Pull request by kirill-korniakov**

### CODE REVIEW

File: code/color_space/src/viewmodel/view_model.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 class ViewModel:
     @staticmethod
     def is_correct(color):
-        return map(operator.methodcaller("isdigit"), color) == [True] * 3
+        return list(map(operator.methodcaller("isdigit"), color)) == [True] * 3
```
Comment (okainov) [inline code/color_space/src/viewmodel/view_model.py:12]: the whole line looks suspicious, why not `return all([isdigit(x) for x in color])` or something similar?

File: .travis.yml [hunk 2]
```diff
@@ -1,15 +1,16 @@
 language: python
 
-python:
-  - "2.7"
+# python:
+  # - "2.7"
   #- "3.5"
```
Comment (okainov) [inline .travis.yml:5]: Probably makes sense to update to 3.8 :)

File: code/color_space/src/viewmodel/view_model.py (hunk 1 above)
Comment (kirill-korniakov) [reply code/color_space/src/viewmodel/view_model.py:12]: @okainov well, this is your line, i've just fixed it a bit ;-) Feel free to submit a PR, will be able to return to it a bit later...

File: .travis.yml (hunk 2 above)
Comment (kirill-korniakov) [reply .travis.yml]: So far relying on whatever TravisCI suggests out of the box. Will need to revisit later with all thee versions for all dependencies...

**Pull request merged**